Ricochet/Assets/_Scripts/Managers/GameManager.cs

```diff
@@ -302,6 +306,8 @@ public void BallGoalCollision(GameObject ball, ETeam team, int points)
             {
                 ball.GetComponent<Ball>().OnBallGoalCollision();
                 ball.SetActive(false);
+                camShake.SetActive(true);
```

Comment:
The Main Camera should always be active. I think this line is redundant. Or is camShake supposed to be the object containing the Camera Shake script? If so, your tooltip is wrong.

Reply:
I will fix the tootip. Thats exactly what it means. It contains the object containing the cam shake script